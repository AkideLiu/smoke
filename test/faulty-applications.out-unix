no-command-file
  There is no command file.
no-inputs
  There are no args or STDIN files.
no-outputs
  There are no STDOUT or STDERR files.
no-such-application
  The application "fixtures/no_such_application" does not exist.
non-executable-application
  The application "fixtures/non_executable_application" is not executable.

5 tests, 5 failures
