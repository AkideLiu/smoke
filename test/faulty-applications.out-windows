no-command-file
  There is no command file.
no-inputs
  There are no args or STDIN files.
no-outputs
  There are no STDOUT or STDERR files.
no-such-application
  The application "fixtures/no_such_application" does not exist.
non-executable-application
  The application "fixtures/non_executable_application" could not be executed.
  fixtures/non_executable_application: createProcess: does not exist (No such file or directory)

5 tests, 5 failures
