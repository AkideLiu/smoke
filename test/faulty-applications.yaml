tests:
  - name: faulty-applications
    args:
      - fixtures/faulty-applications
    exit-status: 1
    stdout:
      - file: io/faulty-applications.out-unix
      - file: io/faulty-applications.out-windows
