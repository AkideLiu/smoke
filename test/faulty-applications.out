no-command-file
  There is no command file for the "no-command-file" test.
no-such-application
  The application "fixtures/no_such_application.rb" does not exist.
non-executable-application
  The application "fixtures/non_executable_application.rb" is not executable.

3 tests, 3 failures
